include/exadg/poisson/user_interface/application_base.h

```diff
@@ -165,10 +165,22 @@ class ApplicationBase
       double AR = calculate_aspect_ratio_vertex_distance(*grid->triangulation, mpi_comm);
       pcout << std::endl << "Maximum aspect ratio (vertex distance) = " << AR << std::endl;
 
-      dealii::QGauss<dim> quad(param.degree + 1);
-      AR =
-        dealii::GridTools::compute_maximum_aspect_ratio(*grid->mapping, *grid->triangulation, quad);
-      pcout << std::endl << "Maximum aspect ratio (Jacobian) = " << AR << std::endl;
+      if(grid->triangulation->all_reference_cells_are_hyper_cube())
+      {
+        dealii::QGauss<dim> quad(param.degree + 1);
```

Comment:
Personally, I would prefer if we could use https://www.dealii.org/developer/doxygen/deal.II/classReferenceCell.html#a56bb8396fda394d712fc4f19de4b02c4 and implement ther logic here and below only once.

Reply:
Yes, like this (slight change `reference_cells.size() == 1`).

Reply:
Addressed in 60211edbe758f3ed0688ae53da0c7410a3104d79.